.dagger/helm.go

```diff
@@ -48,7 +48,7 @@ func (h *Helm) Test(ctx context.Context) error {
 		WithEnvVariable("KUBECONFIG", "/.kube/config").
 		WithEnvVariable("CACHEBUSTER", identity.NewID()).
 		WithExec([]string{"kubectl", "get", "nodes"}).
-		WithExec([]string{"helm", "install", "--wait", "--create-namespace", "--namespace=dagger", "dagger", "."}).
+		WithExec([]string{"helm", "install", "--wait", "--create-namespace", "--namespace=dagger", "--set=engine.image.tag=main", "dagger", "."}).
```

Comment:
Instead of the `engine.image.tag`, we should set the entire image.

Let me whip up that PR - that will be a breaking change, just in time for `v0.13.0` - and then this can be rebased on top of that one.

Reply:
https://github.com/dagger/dagger/pull/8348

Reply:
Rebased to use `engine.image.ref`